internal/builtinfunctions/functions_test.go

```diff
@@ -819,3 +821,113 @@ func Test_floatToFormattedString_success(t *testing.T) {
 		}
 	}
 }
+
+func Test_bindConstants(t *testing.T) {
+	items := []any{
+		map[string]any{"loop_id": 1},
+		map[string]any{"loop_id": 2},
+		map[string]any{"loop_id": 3},
+	}
+	repeatedInputs := map[string]any{
+		"a": "A", "b": "B",
+	}
+	repeatedInputName := builtinfunctions.CombinedObjPropertyConstantName
+	itemName := builtinfunctions.CombinedObjPropertyItemName
+	outputExp := []any{
+		map[string]any{itemName: items[0], repeatedInputName: repeatedInputs},
+		map[string]any{itemName: items[1], repeatedInputName: repeatedInputs},
+		map[string]any{itemName: items[2], repeatedInputName: repeatedInputs},
+	}
+	functionToTest := builtinfunctions.GetFunctions()["bindConstants"]
+	output, err := functionToTest.Call([]any{items, repeatedInputs})
+	assert.NoError(t, err)
+	assert.Equals(t, output.([]any), outputExp)
+
+	// no items in input list
+	output, err = functionToTest.Call([]any{[]any{}, repeatedInputs})
+	assert.NoError(t, err)
+	assert.Equals(t, output.([]any), []any{})
+}
+
+func defaultPropertySchema(t schema.Type) *schema.PropertySchema {
+	return schema.NewPropertySchema(t, nil, false, nil, nil, nil, nil, nil)
+}
+
+func joinStrs(s1, s2 string) string {
+	return strings.Join([]string{s1, s2}, builtinfunctions.CombinedObjIDDelimiter)
+}
+
+// TestHandleTypeSchemaCombine tests the error cases for invalid input, and
+// the equivalence classes of valid input creates the expected behavior in
+// the returned error and returned type and type name, respectively.
```

Comment:
I don't think "equivalence classes" is clear.

Reply:
i removed it